h2o-r/tests/testdir_algos/coxph/runit_coxph_predict.R

```diff
@@ -2,99 +2,153 @@ setwd(normalizePath(dirname(R.utils::commandArgs(asValues=TRUE)$"f")))
 source("../../../scripts/h2o-r-test-setup.R")
 
 
+pred.r <- function(fit.8, tstdata.8) {
```

Comment:
param names indicate this has something to do with cast #8, but I dont think thats the case, samoe for compare.results

Reply:
You are definitely right, fixing this.